src/ai/susi/server/api/aaa/ResendVerificationLinkService.java

```diff
@@ -32,12 +32,13 @@
 /**
  * Created by dravit on 25/7/17.
  * parameter : emailId
- * sample request : 
+ * sample request :
  * http://127.0.0.1:4000/aaa/resendVerificationLink.json?emailId=dravit.lochan@gmail.com
```

Comment:
Email IDs should be set in a global configuration. Please change this accordingly.